public/css/contactUs.css

```diff
@@ -170,7 +175,11 @@ h4 {
         position: relative;
         margin-left: -60vw;
         bottom: 2vh;
-	}
+    }
+
+    .p-e {
+        width: vw;
```

Comment:
It's duplicated here

Reply:
It does nothing. This was supposed to before that media query. But it does nothing. It's not attached to any class